tools/goctl/go.mod

```diff
@@ -13,7 +13,7 @@ require (
 	github.com/spf13/pflag v1.0.5
 	github.com/stretchr/testify v1.8.4
 	github.com/withfig/autocomplete-tools/integrations/cobra v1.2.1
-	github.com/zeromicro/antlr v0.0.1
+	github.com/zeromicro/antlr v1.0.0
```

Comment:
there is no v1.0.0 tag? See https://github.com/zeromicro/antlr/tags

Reply:
goland told me v1.0.0.

Thanks for reminding! Fixed.